src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -13,22 +14,33 @@ object ResultView {
     private const val SEPARATOR = "- "
     private const val TOTAL_RATE_PREFIX = "총 수익률은 "
     private const val TOTAL_RATE_SUFFIX = "입니다."
+    private const val ACCORDANCE_SUFFIX = "개 일치"
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private const val ACCORDANCE_SUFFIX = "%d개 일치"
```